@here/olp-sdk-dataservice-read/lib/client/DataRequest.ts

```diff
@@ -29,73 +29,106 @@ export class DataRequest {
     private version?: number;
     private billingTag?: string;
 
+    /**
+     * Gets a data handle for the request.
+     * 
+     * The data handle identifies a specific blob so that you can get the contents of that blob.
+     * The data handle can only contain alphanumeric characters, '-', and '.' [0-9, a-z, A-Z, -, .].
+     * The maximum length of this field is 1024 characters.
```

Comment:
Please, remove it. We don't have any validations for now

Reply:
Fixed